website/docs/terms/elt.md

```diff
@@ -0,0 +1,139 @@
+- Establish code-based tests
+- Implement CI/CD workflows
+- Document data models like typical software code.
+
+#### Scaling, made sustainable
+
+As your business grows, the number of data sources correspondingly increases along with it. As such, so do the number of transformations and models needed for your business. Managing a high number of transformations without version control or automation is not scalable.
+
+The ELT workflow capitalizes on transformations occurring last to provide flexibility and software engineering best practices to data transformation. Instead of having to worry about how your extraction scripts scale as your data increases, data can be extracted and loaded automatically with a few clicks.
+
+### Bring the power to the people
+
+The ELT workflow opens up a world of opportunity for the people that work on that data, not just the data itself.
+
+#### Empowers data team members
+
+Data analysts, analytics engineers, and even data scientists no longer have to be dependent on data engineers to create custom pipelines and models. Instead, they can use point-and-click products such as Fivetran and Airbyte to extract and load the data for them. 
+
+Having the transformation as the final step in the ELT workflow also allows data folks to leverage their understanding of the data and SQL to focus more on actually modeling the data. 
+
+#### Promotes greater transparency for end busines users
+
+Data teams can expose the version-controlled code used to transform data for analytics to end business users by no longer having transformations hidden in the ETL process. Instead of having to manually respond to the common question, “How is this data generated?” data folks can direct business users to documentation and repositories. Having end business users involved or viewing the data transformations promote greater collaboration and awareness between business and data folks.
+
+## ELT tools
+
+As mentioned earlier, the recent development of certain technologies and products has helped lower the barrier of entry to implementing the ELT workflow. Most of these new products act as one or two parts of the ELT process, but some have crossover across all three parts. We’ll outline some of the current tools in the ELT ecosystem below.
+
+| Product | E, L, or T? | Description | Open-source option? |
+|---|---|---|---|
+| Fivetran/HVR | E, some T, L | Fivetran is a SaaS company that helps data teams extract, load, and perform some transformation on their data. Fivetran easily integrates with modern data warehouses and dbt. They also offer transformations that leverage dbt Core. | :x: |
+| Stitch by Talend | E, L | Stitch (part of Talend) is another SaaS product that has many data connectors to extract data and load it into data warehouses. | :x: |
+| Airbyte | E, L | Airbyte is an open-source and cloud service that allows teams to create data extraction and load pipelines. | :white_checl_mark: |
```

Comment:
For Airbyte, think we're looking for :white_check_mark: vs :white_checl_mark:

Reply:
Emojis strike again 😢 Fixed this in the latest commit.